manual/googlechrome-extensions/googlechrome-zenhub/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,9 @@
+$bits = Get-ProcessorBits
+$packageName = 'googlechrome-zenhub'
```

Comment:
since `$packageName` isn't used anywhere, probably no need to set this parameter